lab_1_classify_by_unigrams/main.py

```diff
@@ -64,6 +101,12 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if isinstance(predicted, list) and isinstance(actual, list):
```

Comment:
return None here

Reply:
I have "return None" at the end of the cycle. I tried to place it in a different way, but got mistakes with returns (there was something like  "you cannot return value and None").

Reply:
.

Reply:
done